tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -442,6 +451,37 @@ type ProjectWorkflowBuilder(initialProject: SyntheticProject, ?checker: FSharpCh
             return { ctx with Signatures = ctx.Signatures.Add(fileId, newSignature) }
         }
 
+    /// Find a symbol using the provided range, mimicing placing a cursor on it in IDE scenarios
+    [<CustomOperation "placeCursor">]
+    member this.PlaceCursor(workflow: Async<WorkflowContext>, fileId, line, colAtEndOfNames, fullLine, symbolNames) = 
+        async {
+            let! ctx = workflow
+            let! results = checkFile fileId ctx.Project checker
+            let typeCheckResults = getTypeCheckResult results
+
+            let su = typeCheckResults.GetSymbolUseAtLocation(line,colAtEndOfNames,fullLine,symbolNames)
+
+            return {ctx with Cursor = su}
+        }
+
+
+
+    /// Find all references within a single file, results are provided to the 'processResults' function
+    [<CustomOperation "findAllReferences">]
+    member this.FindAllReferences(workflow: Async<WorkflowContext>, fileId: string,  processResults) =
```

Comment:
A bit of nitpicking here: I'd suggest either calling this `FindAndProcessReferences`, or keeping the name and only finding references, or maybe building in assertions right here and calling this `AssertAllReferences`. 

You know, just matching the name to the actual behavior :) Otherwise thanks for extending the testing framework, that is useful.

Reply:
It is called after the VS action, to make it obvious which IDE action this is supposed to mimic.